devices/sam/same70_s70_v70_v71.xml

```diff
@@ -54,19 +168,26 @@
       <vector position="27" name="TC4"/>
       <vector position="28" name="TC5"/>
       <vector position="29" name="AFEC0"/>
-      <vector position="30" name="DACC"/>
+      <vector device-series="e70|s70" device-pin="n" device-flash="21" device-variant="a" position="30" name="DACC"/>
+      <vector device-series="e70|s70|v70" device-pin="j|n|q" device-flash="19|20" device-variant="a|b" position="30" name="DACC"/>
+      <vector device-series="e70|s70|v71" device-pin="j|n|q" device-flash="19|20|21" device-variant="b" position="30" name="DACC"/>
+      <vector device-series="e70|s70|v71" device-pin="j|q" device-flash="20|21" device-variant="a" position="30" name="DACC"/>
+      <vector device-series="v71" device-pin="n|q" device-flash="19" device-variant="a" position="30" name="DACC"/>
```

Comment:
That data looks quite strange but some devices of revision A really seem to have no DAC. This is contradictory to all datasheets and errata documents I could find, even the old ATMEL ones. But the device headers agree.

For SAMV71N rev A only the smallest flash size has a DAC, for SAMV71J rev A it is the other way around :confused: